unic/ucd/age/src/tables/age_values.rsv

```diff
@@ -1,4 +1,4 @@
-// WARNING: Auto-generated by `tools/gen_ucd_tables.py`. DO NOT EDIT MANUALLY!
+// WARNING: Auto-generated by unic-gen. DO NOT EDIT MANUALLY!
```

Comment:
For someone new to UNIC, this could be a bit unclear. Can we please use this?
```
// WARNING: Auto-generated by `unic-gen` crate. DO NOT EDIT MANUALLY!
```

(I know it's not introduced in this diff, but I just noticed it since it's going in use.)